tests/test_endpoints/endpoint_data.py

```diff
@@ -301,29 +303,30 @@
             'admin2_name',
             'admin1_ref',
         ],
-    },
-    '/api/resource': {
+    },  # (hdx_id, dataset_hdx_id, name, format, update_date, is_hxl, download_url, hapi_updated_date)
```

Comment:
is the commented line here on purpose ?

Reply:
I think it is a remnant of the process I used to make the file, so it can be deleted.